pkgs/development/libraries/science/math/mkl/default.nix

```diff
@@ -43,16 +30,23 @@ stdenvNoCC.mkDerivation rec {
   '' else ''
     rpmextract rpm/intel-mkl-common-c-${date}-${rel}-${date}-${rel}.noarch.rpm
     rpmextract rpm/intel-mkl-core-rt-${date}-${rel}-${date}-${rel}.x86_64.rpm
+    rpmextract rpm/intel-openmp-19.0.0-${rel}-19.0.0-${rel}.x86_64.rpm
```

Comment:
This is the meat of the change: we're just unpacking this extra RPM and putting it next to `libmkl_rt.so` so it has its dependencies available.  Since this comes in the same src tarball, we can maintain this package as an efficient fixed-output derivation while still keeping the nix expression relatively simple.